test/Table_Tests/src/IO/Fetch_Spec.enso

```diff
@@ -457,6 +457,30 @@ add_specs suite_builder =
                     HTTP.fetch headers=headers2 uri
                     get_num_response_cache_entries . should_equal 2
 
+        group_builder.specify "Does not attempt to make room for the maximum file size when that is larger than the total disk size" pending=pending_has_url <| Test.with_retries <|
```

Comment:
Test name seems misleading.
Where are we mocking disk size in this test? What I see is we are overriding **total cache size**, not disk space.

Reply:
Yes -- changed to "total cache size".